PulseSampleApp/PulseSampleApp/Info.plist

```diff
@@ -26,6 +26,11 @@
 	<string>LaunchScreen</string>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Hi, @rcvrgs . Pulse works with https but the video content (the Cleopatra video) is streamed via http. That's why I enabled it for the sample app.